pynest/nest/tests/test_getconnections.py

```diff
@@ -28,8 +28,11 @@
 
 nest.set_verbosity('M_ERROR')
 
+HAVE_GSL = nest.ll_api.sli_func("statusdict/have_gsl ::")
+
 
 @nest.ll_api.check_stack
+@unittest.skipIf(not HAVE_GSL, 'GSL is not available')
```

Comment:
I'm not sure why this is necessary, I've probably missed something... couldn't we use only models that do not require GSL? I don't think invalid models would appear in ``Models`` if they were not compiled, would they?

If gap junction need to be tested specifically, then using an ``if HAVE_GSL else`` with different models for hh_psc_alpha_gap?

Reply:
You are right, I haven't used `label` for synapses before, so I just followed an example that use `hh_psc_alpha_gap`. Have switched to normal `iaf_psc_alpha` and removed the `HAVE_GSL`, and the tests pass.